Azure/sonic-utilities
# ZTP CLI commands

Implemented following commands to use Zero Touch Provisioning

show ztp status
config ztp enable
config ztp disable
config ztp run

Signed-off-by: Rajendra Dendukuri <rajendra.dendukuri@broadcom.com>



**- What I did**
Added following CLI commands for Zero Touch Provisioning feature. 

show ztp status
config ztp enable
config ztp disable
config ztp run

If sonic-ztp package is not included in the image, ztp cli commands display following message.
"ZTP feature unavailable in this image version"

**- How I did it**
Modified config/main.py and show/main.py

**- How to verify it**
Execute individual commands and check ZTP flow

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

-->
